include/ipvs/conn.h

```diff
@@ -280,6 +281,36 @@ static inline void dp_vs_control_add(struct dp_vs_conn *conn, struct dp_vs_conn
     rte_atomic32_inc(&ctl_conn->n_control);
 }
 
+static inline bool
+dp_vs_conn_is_template(struct dp_vs_conn *conn)
+{
+    return  (conn->flags & DPVS_CONN_F_TEMPLATE) ? true : false;
+}
+
+static inline void
+dp_vs_conn_set_template(struct dp_vs_conn *conn)
+{
+    conn->flags |= DPVS_CONN_F_TEMPLATE;
+}
+
+static inline bool
+dp_vs_conn_is_in_timer(struct dp_vs_conn *conn)
+{
+    return (conn->flags & DPVS_CONN_F_IN_TIMER) ? true : false;
+}
+
+static inline void
+dp_vs_conn_set_in_timer(struct dp_vs_conn *conn)
+{
+    conn->flags |= DPVS_CONN_F_IN_TIMER;
+}
+
+static inline void
+dp_vs_conn_clear_in_timer(struct dp_vs_conn *conn)
+{
+    conn->flags &= ~DPVS_CONN_F_IN_TIMER;
+}
+
```

Comment:
What's the purpose to use the conn flag "DPVS_CONN_F_IN_TIMER"？ Is there any problem if the flag not used？

Reply:
The purpose is for easy code logic and it is similar as the processing of DPVS_CONN_F_HASHED.